.github/ISSUE_TEMPLATE/8_design_doc.yml

```diff
@@ -0,0 +1,32 @@
+name: Submit a Flutter design document.
+description: As a contributor, I would like to submit a design document.
```

Comment:
here and below, how about instead of framing this as a "submission" we frame it more like someone is filing it for archival purposes, or some such... not sure how to phrase it... The thing I'm trying to get away from is any implication that this is a formal process of review, as opposed to just a tool to help people get more eyes on their docs. Maybe "publish"?

"As a contributor, I would like to publish a design document."

Reply:
I think the appropriate verbiage that would meet the vision you see is something like:

"As a contributor, I would like to share a design document."